src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $color-grey;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    color: $color-main-accent;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__product-code,
+  &__review-value {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    color: $color-secondary;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-value {
```

Comment:
better don't do it, it can be confusing, like I can think that review is a block